test-scripts/local-is-deployment/README.md

```diff
@@ -0,0 +1,22 @@
+## Executing local Identity Server Deployment
+
+### Prerequisites
+1. Inorder to execute the Testgrid executable you should have a mysql setup running locally with username/password, root/root.
+2. The local machine should have curl and unzip installed.
+3. Java should be installed and configured. JAVA_HOME env variable must be set.
```

Comment:
Product zip  also need to be copied, right?

Reply:
This is added to the read me